recipe_scrapers/leanandgreenrecipes.py

```diff
@@ -0,0 +1,51 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+
+class LeanAndGreenRecipes(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "leanandgreenrecipes.net"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        instructions = (
+            self.soup.find("div", {"class": "item-list"}).find_next("ol").find_all("li")
+        )
+        return "\n".join(
+            [normalize_string(instruction.get_text()) for instruction in instructions]
+        )
+
+    def ratings(self):
+        return self.schema.ratings()
+
+    def cuisine(self):
+        cuisine = self.schema.cuisine()
+        return cuisine[cuisine.find(">") + 1 :].replace("</a>", "")
+
+    def description(self):
+        return (
+            self.schema.description()
+            .replace("</p> <p>", "\n")
+            .replace("<p>", "")
+            .replace("</p>", "")
+        )
```

Comment:
Out of curiosity: are you able to find any counter-example recipes that _do not_ require this kind of escaping of the schema description (and/or `recipeCuisine` field)?

Reply:
thanks for reviewing, provided a fix, please check.

: wonder how  to add the `hacktoberfest-accepted` label into this merge request?

Reply:
I'll remember you @wengtad and will attach `hacktoberfest-accepted` labels to your PRs through October 😉

Reply:
Thanks!